configs/projects/client-tools-runtime-master.rb

```diff
@@ -0,0 +1,4 @@
+project 'client-tools-runtime-master' do |proj|
+  # Common settings
```

Comment:
Can we stop building kearney and johnson?

Reply:
I don't know. We have platerprise steering next week, I'll ask the folks in there if we can get some clear consensus/communication about the state of all the PE branches.

For now I was just going to leave them